example/example.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flutter/material.dart';
 import 'package:generic_dropdown_widget/generic_dropdown_widget.dart';
 
-class Dropdown extends StatelessWidget {
+final class Dropdown extends StatelessWidget {
```

Comment:
final? no subclassing is possible after that change. this would make it impossible to extend the dropdown further down the line.